docs/repo/property-page-architecture.drawio.svg

```diff
@@ -0,0 +1,234 @@
+<svg host="65bd71144e" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" width="361px" height="441px" viewBox="-0.5 -0.5 361 441" content="&lt;mxfile host=&quot;9e9f10d4-3a71-4708-980f-a763b3b7768b&quot; modified=&quot;2020-10-30T17:43:04.531Z&quot; agent=&quot;5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Code/1.50.1 Chrome/83.0.4103.122 Electron/9.2.1 Safari/537.36&quot; version=&quot;13.6.5&quot; etag=&quot;olyi-gRlhe6lnlsRP_5v&quot;&gt;&lt;diagram id=&quot;--xkG6ouKXTrO3x_hQlo&quot; name=&quot;Page-1&quot;&gt;5Zhbk9ogGIZ/jZftQDAeLjXutp3pztg6bfeWETS0JF9KiIf++oISjRI7O7OJ2tabhBcw8Lwfxw6Jks07RbP4CRiXnQCxTYdMOkGAQ9w1D6tsnYLDYK8slWBOOwoz8Ys7ETm1EIznJwU1gNQiOxXnkKZ8rk80qhSsT4stQJ5+NaNL7gmzOZW++k0wHe/VLkJH/T0Xy9h9OSwzElqWdUIeUwbrikQeOiRSAHr/lmwiLi28Esu+3uOF3EO7FE/1Syo47Csqi7Jrb036ywfXPL0tu6ygSBm31VCHjNex0HyW0bnNXRuXjRbrRJoUNq8LIWUEEtSuLulRhBYLo+dawQ9eySF9hKLI1oBUV/TF7md01z6uNN9c7CM+kDMhxyHhWm1NkU1py77G9jS5PhpHSjviimllOepiZXn44yNP8+KQ1uMNPbyBxTtV8N2GZYA+FXzX1NH0HyY+uCLxnke8+wfiSCSZ5InpFtUC0nuxwMIXZrYZSbFMTV4iGLNNasEb3L+iOX3PHGLNGVs+ymA27LlaibmZ2htzIoXUkmM0j3fVcTMYgzOMpAZjrwYjaQDjwMMYWowzTVNGlcX40wU5o5rabilYmQVTNYj1rwnww56h4gwOWwrwoedM380+menSttxZXMkghELEaw1CCE/GtzIoQC9wqG7sNOFQ+a2KRT1r0VeqBBT50ZIcCtXGRHSRbNMz1Pk4CK84DjD2KA8qA0FYrijf5pon/8OU5EV8nRetRTzxvMC7Tf7z6OmjeXwuZJNh/rp5R4HbiZHJEDUE/2wgkJqlGre14ylPuhX4w7O1OqssDve2TrdgxyHOb2KHfyDDyPoRFbmG5Bpu3NngQLd0wz+sRdOZERTP4F6GQPMrMRnWMEZtMfbPXAx0ys1R+NEEOLen4jflStw8+FdEe/PguzVXP02BN8njrd0ur3L3SR5+Aw==&lt;/diagram&gt;&lt;/mxfile&gt;">
```

Comment:
How did you generate the SVG?

Reply:
I'm guessing it can be opened in https://app.diagrams.net/ (part of https://drawio-app.com/)

Reply:
That's correct. Specifically, I used the "Draw.io Integration" extension for VS Code. And by saving it as a .drawio.svg file, I can embed it directly into the .md file.